ramls/dataImport.raml

```diff
@@ -176,6 +177,15 @@ resourceTypes:
               body:
                 text/plain:
                   example: "Internal server error"
+  /jobExecutions/{jobExecutionId}:
+    /downloadUrl:
```

Comment:
please, add the newly implemented endpoint to the moduleDescriptorTemplate.json file